Detections/OracleDatabaseAudit/OracleDBAuditNewUserDetected.yaml

```diff
@@ -0,0 +1,36 @@
+id: cca7b348-e904-4a7a-8f26-d22d4d477119
+name: OracleDBAudit - New user account
+description: |
+  'Detects when an action was made by new user.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+queryFrequency: 3h
+queryPeriod: 3h

```

Comment:
This should match the lookback period, 14d max

Reply:
updated